resources/views/inputs/tile-selection.blade.php

```diff
@@ -8,6 +8,12 @@
     'single' => false,
     'hiddenOptions' => false,
     'mobileShowRows' => 3,
+    'withoutTitle'  => false,
+    'withIcon' => false,
```

Comment:
Shouldn't this default to true since that was the default before? I think a `withoutIcon` may be more appropriate here